khmer/_khmer.cc

```diff
@@ -145,7 +158,7 @@ static
 PyObject *
 Read_get_name(khmer_Read_Object * obj, void * closure )
 {
-    return PyBytes_FromString(obj->read->name.c_str()) ;
+    return PyUnicode_FromString(obj->read->name.c_str()) ;
```

Comment:
This is a design decision: All strings passed to Python will be Unicode, but we accept either Unicode or Bytes as parameters for all exposed C++ methods.